### DISCUSSION TITLE
Stamp positioning/orientation on the page

### DISCUSSION THREAD

**Discussion by eduperottoni** [MatthiasValvekens/pyHanko#276] [Q&A]
Hi,

First of all, thank you for this amazing job!
I was trying to find a way to insert the signature stamp in vertical orientation with pyHanko . 

At the moment, I use `TextStampStyle` and `StaticStampStyle` to produce stamps in horizontal orientation, as well as the examples shown in the documentation. Now I want to position the stamp in the left or right side of the page, with the text box oriented by the y axis. The figure below shows an example of what I'm trying do to:

![ksnip_20230628-144720](https://github.com/MatthiasValvekens/pyHanko/assets/69221498/2b63f60c-162e-4796-a0fb-829bb221e982)

In the documentation, there is no example related with that and reading about the stamp style and signature field options, it seems that the library doesn't support it yet.  Am I right or did I miss something?

**Comment by MatthiasValvekens** [comment]
Hi @eduperottoni,


Sorry for the delay in replying. PyHanko supports vertical text to some extent, but it's mostly intended for East-Asian languages. I think the easiest way to achieve what you want is to run the layout logic the usual way, and then rotate the content by 90 degrees.

This currently isn't supported out of the box, but something akin to #266 can probably be leveraged to make this work. Feel free to add a comment there :).
